types/image.go

```diff
@@ -60,7 +60,7 @@ type ArtifactReference struct {
 
 // ArtifactInfo is stored in cache
 type ArtifactInfo struct {
-	SchemaVersion int
+	Version       int
```

Comment:
I think we should leave a comment about when we have to update the version.

Reply:
What do you propose?